vault/token_store.go

```diff
@@ -1396,13 +1447,36 @@ func (ts *TokenStore) tokenStoreRoleCreateUpdate(
 		entry.Period = time.Second * time.Duration(data.Get("period").(int))
 	}
 
+	var resp *logical.Response
+
+	explicitMaxTTLInt, ok := data.GetOk("explicit_max_ttl")
+	if ok {
+		entry.ExplicitMaxTTL = time.Second * time.Duration(explicitMaxTTLInt.(int))
+	} else if req.Operation == logical.CreateOperation {
```

Comment:
The reason it's like that is so that if we decide to set a default value later, it will automatically "just work" by modifying the schema, without having to go through code and make sure we're honoring it.